frontend/src/container/TriggeredAlerts/Filter.tsx

```diff
@@ -51,6 +75,7 @@ function Filter({
 
 	const options = uniqueLabels.map((e) => ({
 		value: e,
+		title: '',
```

Comment:
To stop the default title from showing up. ( antd uses rc-select under the hood )
https://github.com/react-component/select/blob/ec0c6c748a24aa500ede15f1d2aacd0dacf4f63b/src/OptionList.tsx#L345